doi/src/main/webapp/paths/doi-instance.yaml

```diff
@@ -33,7 +33,7 @@ get:
   tags:
     - "instances"
   summary: "Get a list of DOI statuses"
-  description: "List the statuses of all DOIs created by the user."
+  description: "List the statuses of all the DOIs which User has access."
```

Comment:
Maybe something like 'Return the DOIStatus for all DOI's the User has permission to view'